avogadro/rendering/glrenderer.cpp

```diff
@@ -169,16 +204,43 @@ void GLRenderer::setTextRenderStrategy(TextRenderStrategy* tren)
 void GLRenderer::applyProjection()
 {
   float distance = m_camera.distance(m_center);
+#ifdef _3DCONNEXION
```

Comment:
I think I would rather test this as a general change rather than a bunch of `#ifdef` here.